tests/dashbio_demos/app_forna_container.py

```diff
@@ -0,0 +1,585 @@
+import os
+
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_daq as daq
+import dash_bio
+
+# running directly with Python
+if __name__ == '__main__':
+    from utils.app_standalone import run_standalone_app
+
+# running with gunicorn (on servers)
+elif 'DASH_PATH_ROUTING' in os.environ:
+    from tests.dashbio_demos.utils.app_standalone import run_standalone_app
+
+
+def header_colors():
+    return {
+        'bg_color': '#85002D',
+        'font_color': 'white'
+    }
+
+
+initial_sequences = {
+    'PDB_01019': {
+        'sequence': 'AUGGGCCCGGGCCCAAUGGGCCCGGGCCCA',
+        'structure': '.((((((())))))).((((((()))))))',
+        'options': {
+            'applyForce': True,
+            'circularizeExternal': True,
+            'avoidOthers': True,
+            'labelInterval': 5,
+            'name': 'PDB_01019'
+        }
+    }
+}
+
+
+def description():
+    return 'RNA secondary structure analysis.'
+
+
+def layout():
+    return html.Div(
+        id='forna-body',
+        className='app-body',
+        children=[
+            html.Div(
+                id='forna-control-tabs',
+                className='control-tabs',
+                children=[
+                    dcc.Tabs(id='forna-tabs', value='what-is', children=[
+                        dcc.Tab(
+                            label='About',
+                            value='what-is',
+                            children=html.Div(className='control-tab', children=[
+                                html.H4(className='what-is', children='What is FornaContainer?'),
+                                html.P('FornaContainer is a force-directed graph that is '
+                                       'used to represent the secondary structure of nucleic '
+                                       'acids (i.e., DNA and RNA).'),
+                                html.P('In the "Add New" tab, you can enter a sequence '
+                                       'by specifying the nucleotide sequence and the '
+                                       'dot-bracket representation of the secondary '
+                                       'structure.'),
+                                html.P('In the "Sequences" tab, you can select which '
+                                       'sequences will be displayed, as well as obtian '
```

Comment:
```suggestion
                                       'sequences will be displayed, as well as obtain '
```

could this section be simplified with `dcc.Markdown`?

Reply:
Yes, good point! I used it in this app, but I'm thinking that we should update the rest of them too. I'll open an issue for that.